### PR TITLE
Fix #318: Add promoting releases to the community to dev docs

### LINKED ISSUE #318: Document to notify mailing list after releases
**Is your feature request related to a problem? Please describe.**

The maintenance docs (section [Releases](https://hexatomic.github.io/hexatomic/dev/v0.6/maintenance/releases/index.html)) should mention to (and how to) notify the mailing list after releases, ideally with a template for the email.

### PR DESCRIPTION
**Pull request by stephandruskat**
## Pull request type

 

- Documentation content changes

## What is the current behavior?

Issue Number: https://github.com/hexatomic/hexatomic/issues/318

## What is the new behavior?

- Add a section on notifying the community of new releases to the maintenance documentation
- Mention mailing list usage in takeover notes for new maintainers
- Add mailing list to maintenance task list

## Does this introduce a breaking change?

- No

## Other information

To review this PR, please check the SonarCloud quality summary on my branch first.
You will have to look at the one-to-last commit, as the last commit removes the sonar properties I'm using for my branch (which won't work for you): https://github.com/stephandruskat/hexatomic/runs/2932672615.

## Pull request checklist

Please check if your PR fulfills the following requirements:
- [x] Tests for the changes have been added **or** the PR is neither a bug fix nor a feature
- [x] Docs have been reviewed and added / updated if needed **or** the PR is neither a bug fix nor a feature
- [x] The *Unreleased* section in CHANGELOG.md has been amended to reflect the changes in this PR if needed
- [x] Build (`mvn verify`) was run locally and any changes were pushed
- [x] The pull request is against the correct branch (`master` for bug fixes, `develop` for new functionality)
- [x] [Dependencies and citation templates](https://github.com/hexatomic/hexatomic/tree/develop/releng/templates) have been updated where necessary **or** there are no new dependencies

---

# Checklist for maintainers

## Releases

- Do **NOT** push the green merge button on GitHub.  
Follow the latest version of the [developer/maintainer docs](https://hexatomic.github.io/hexatomic/dev/) for making releases.
- Check that license and citation information for dependencies are complete (each dependency has a folder in `/THIRD-PARTY/`).

### CODE REVIEW

**Comment by stephandruskat** [comment]
Hello maintainers, I have tried to set up SonarCloud for my repository, so that the checks should be run on my repository.

File: docs/dev/src/maintenance/releases/README.md [hunk 1]
```diff
@@ -71,6 +71,28 @@ You can now make a release from this tag:
 
 ![Animation showing how to create a release on GitHub](release.gif)
 
+## Promoting releases to the community
+
+Hexatomic has a mailing list for users: `hexatomic-users@lists.hu-berlin.de`.
+The maintainers of the Hexatomic software project are also [maintainers of this mailing list](TODO).
```
Comment (bbunzeck) [inline docs/dev/src/maintenance/releases/README.md:77]: @stephandruskat 
Is the (TODO) link intended or did you just forget adding the right link?

Comment (sdruskat) [reply docs/dev/src/maintenance/releases/README.md:77]: Good catch, thanks! No, it isn't.

```suggestion
The maintainers of the Hexatomic software project are also [maintainers of this mailing list](https://sympa.cms.hu-berlin.de/sympa/info/hexatomic-users).
```

**Review by bbunzeck — APPROVED** [review]
Everything seems adequate!

**Comment by sdruskat** [comment]
> Everything seems adequate!

Thanks for the review!

**Pull request merged**